lib/forever/cli.js

```diff
@@ -359,7 +359,7 @@ app.cmd('stopall', cli.stopall = function () {
     if (processes) {
       forever.log.info('Forever stopped processes:');
       processes.split('\n').forEach(function (line) {
-        forever.log.data(line);
+        forever.log.warn(line);
```

Comment:
I don't really know if the 'data' level would be considered closer to the warn/error end or the info/verbose end so I just guessed.

Reply:
I think it's neither, it used to be this thing: https://github.com/winstonjs/winston/tree/2.x#using-winston-in-a-cli-tool

Can't find anything equivalent in Winston 3. Can you check how different output is between `.data` in old version and `warn` in new?

Reply:
In Winston 3 it should be possible to define custom log levels, I believe l, if really necessary :)

Reply:
@DABH yeeeeeah, but I wonder what was special about `data` one and how we can replicate it :D

Reply:
Sadly such an old Winston version predates my involvement in the project 😢